inst/include/Rcpp/date_datetime/Datetime.h

```diff
@@ -68,6 +68,20 @@ namespace Rcpp {
 
         operator double()  const { return m_dt; }
 
+        inline std::string format(const char *fmt = "%Y-%m-%d %H:%M:%S") const {
+            char txtiso[64], txtsec[64];
+            time_t t = static_cast<time_t>(std::floor(m_dt));
+            struct tm temp = *localtime(&t); // localtime, not gmtime
+            ::strftime(txtiso, 63, fmt, &temp);
+            ::snprintf(txtsec, 63, "%s.%06d", txtiso, m_us);
+            return std::string(txtsec);
+        }
+
+        inline std::ostream &operator<<(std::ostream & s) const {
```

Comment:
No, do a `ag operator\<\<` inside the repo directory.  All `operator<<` go to ostream from which they get redirected.

But @dcdillon just reminded me that I can't do this as a member function.  So it was a null-op ;-)